processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -156,8 +156,8 @@ public boolean isPresenceCheckMethod(ExecutableElement method) {
     @Override
     public String getPropertyName(ExecutableElement getterOrSetterMethod) {
         String methodName = getterOrSetterMethod.getSimpleName().toString();
-        if ( methodName.startsWith( "is" ) || methodName.startsWith( "get" ) || methodName.startsWith( "set" ) ) {
-            return IntrospectorUtils.decapitalize( methodName.substring( methodName.startsWith( "is" ) ? 2 : 3 ) );
+        if ( methodName.startsWith( "get" ) || methodName.startsWith( "set" ) ) {
```

Comment:
perhaps you told me before, but what if we have a boolean target accessor? e.g. `boolean isAlive()`? Will MapStruct recognise `alive` as property name?

Reply:
When we have accessor it would work. As it is not a setter. However, if you use `isAlive` for your setter then the name of the property would be `isAlive`, which I think is fine, you won't use such name for a setter.